src/gateway.rs

```diff
@@ -80,8 +80,9 @@ impl Gateway {
 
     async fn handle_udp_event(&mut self, logger: &Logger, event: Event) -> Result {
         match event {
-            Event::UnableToParseUdpFrame(buf) => {
-                info!(logger, "ignoring semtech udp parsing error for {:?}", buf)
+            Event::UnableToParseUdpFrame(e, buf) => {
+                info!(logger, "ignoring semtech udp parsing error {e}");
+                info!(logger, "ignoring semtech udp parsing error for {buf:?}");
```

Comment:
is there a reason to not combine this with the line above?

Reply:
No that's a good point. Fixing now